pkg/cloud/services/compute/instance.go

```diff
@@ -575,8 +575,13 @@ func (s *Service) InstanceDelete(machine *clusterv1.Machine) error {
 	if err != nil {
 		return err
 	}
+	if err = deleteInstance(s, parsed.ID()); err != nil {
+		record.Warnf(openStackMachine, "FailedDeleteServer", "Deleted server %s: %v", parsed.ID(), err)
```

Comment:
I think the message should be like in the other case `Failed to delete server....`

Reply:
Fixed.